src/examples/design/color/BackgroundVariables.tsx

```diff
@@ -0,0 +1,189 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+import {
+  ColorParameters,
+  DEFAULT_THEME,
+  getColor,
+  IGardenTheme
+} from '@zendeskgarden/react-theming';
+import { Table } from '@zendeskgarden/react-tables';
+import { Code, LG } from '@zendeskgarden/react-typography';
+import { Markdown } from 'components/MarkdownProvider/components/Markdown';
```

Comment:
I realize these won't compile for CodeSandbox. But I'm not too concerned for a few reasons:
- codesandbox will not be the typical path for these design section examples
- Garden wants to move this `Markdown` component into `@zendeskgarden/react-typography` – at which point we can swap out these imports